Library/Homebrew/dev-cmd/generate-cask-api.rb

```diff
@@ -0,0 +1,62 @@
+# typed: true
+# frozen_string_literal: true
+
+require "cli/parser"
+require "cask/cask"
+
+module Homebrew
+  extend T::Sig
+
+  module_function
+
+  sig { returns(CLI::Parser) }
+  def generate_cask_api_args
+    Homebrew::CLI::Parser.new do
+      description <<~EOS
+        Generates Cask API data files for formulae.brew.sh.
+      EOS
+
+      named_args :none
+    end
+  end
+
+  JSON_TEMPLATE = <<~EOS
+    ---
+    layout: cask_json
+    ---
+    {{ content }}
+  EOS
+
+  def html_template(title)
+    <<~EOS
+      ---
+      title: #{title}
+      layout: cask
+      ---
+      {{ content }}
+    EOS
```

Comment:
The bit of Jekyll HTML template here feels really weird (particularly since this is not really the API itself) and am not a fan of it here. If it's really the only way to do it though then fair enough.

Reply:
@Bo98 Again: game to not do this. It's slightly nicer than a `.gsub` to make this a function, IMO.